bin/dipy_det_track

```diff
@@ -3,7 +3,7 @@
 from __future__ import division, print_function
```

Comment:
A design comment. Since there will be other tracking bin soon in dipy, It might be interesting to have all of them grouped together by naming them `dipy_track_*`, this one being `dipy_track_det`. 

You just removed the `_reconst_` part for dti, csd, cas, so I might have missed that discussion... :)